core/Cargo.toml

```diff
@@ -6,13 +6,9 @@ edition = "2021"
 [dependencies]
 serde = { version = "1.0", default-features = false }
 rand = "0.8.5"
-rug = "1.26.1"
```

Comment:
So, the reason I stopped using `rug` is relevant to `risc0`. It prevents it from compiling properly.